firewall-examples.md

```diff
@@ -77,15 +70,15 @@ $ sdc-enablefirewallrule 0b3adeaf-cfd9-4cbc-a566-148f569c050c
 </pre>
 </div></div>
 
-<p>You can see that both of these rules allow SSH traffic from the Internet. If there is more than one rule that affects incoming traffic, the least restrictive one applies. In this case, the rule that allows SSH traffic to all instances in the datacenter is applied.</p>
+You can see that both of these rules allow SSH traffic from the Internet. If there is more than one rule that affects incoming traffic, the least restrictive one applies. In this case, the rule that allows SSH traffic to all instances in the datacenter is applied.
 
-<p>If you were to disable that rule, however, only the second rule would apply.</p>
+If you were to disable that rule, however, only the second rule would apply.
 
 
 
-<h1><a name="CloudFirewallExamples-AllowHTTPTraffic"></a>Allow HTTP Traffic</h1>
+#[Allow HTTP Traffic](CloudFirewallExamples-AllowHTTPTraffic)
```

Comment:
The previous syntax was for what's called a named link, which creates an anchor in the text. This is better achieved in pure html by putting an `id` on the element.

This isn't documented in the materials I gave you, but in our world we can add the anchor by adding it in curly braces after the heading. As so:

```
### Can I buy support? {#support}
```